TheraLang.BLL/Services/DonationService.cs

```diff
@@ -22,7 +22,8 @@ public DonationService(IUnitOfWork unitOfWork)
             _unitOfWork = unitOfWork;
         }
 
-        public LiqPayCheckoutDto GetLiqPayCheckoutModel(string donationAmount, int? projectId, HttpContext context)
+        public async Task<LiqPayCheckoutDto> GetLiqPayCheckoutModelAsync(string donationAmount, int? projectId,
+            HttpContext context)
```

Comment:
to remove this --> This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
 --> if you want we can wrap up calls into a task and await it